File: exo3/exo3.php
<?php
$word = $argv[1];
$char = $argv[2];
$result = 0;
$i = 0;
while($i < strlen($word)){
	if($word[$i] == $char){
		$result++;
	}
$i++;
}
echo "Le nombre de " .$char. " est : " .$result."\r"."\n";
?>

File: exo3/exo3.py
import sys

word = sys.argv[1]
char = sys.argv[2]
result = 0
i = 0

while i < len(word) :
	if word[i] == char :
		result += 1
	i += 1
print ("Le nombre de " + str(char) + " est : " + str(result))
